checker/src/main/java/org/checkerframework/checker/nullness/CollectionToArrayHeuristics.java

```diff
@@ -35,26 +43,53 @@
  *       </ol>
  * </ol>
  *
- * Note: The nullness of the returned array doesn't depend on the passed array nullness.
+ * <p>Additionally, when the lint option {@link NullnessChecker#LINT_TRUSTARRAYLENZERO} is provided,
+ * a call to {@link Collection#toArray(Object[]) Collection.toArray(T[])} will return an array with
```

Comment:
The option doesn't change what is returned.  It changes the checker's estimate of the return type.  Could you clarify the text?